Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -175,6 +177,8 @@ export function extend(publicAPI, model, initialValues = {}) {
     'imageSampleDistance',
     'maximumSamplesPerRay',
     'autoAdjustSampleDistances',
+    'initialInteractionScale',
+    'interactionSampleDistanceFactor',
```

Comment:
When interacting, VTK changes sample distance and VTK.js changes viewport size
Instead of completely reworking VTK.js approach, I added this setting inspired by VTK's approach
Changing both viewport size and sample distance dynamically would be a lot more complicated

Reply:
gotcha, good with me.